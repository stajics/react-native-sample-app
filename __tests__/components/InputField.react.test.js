import 'react-native';
import React from 'react';
import InputField from '../../app/components/InputField.react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'; // eslint-disable-line

it('renders input field', () => {
  const inputField = renderer.create(
    <InputField />
  ).toJSON();
  expect(inputField).toMatchSnapshot();
});

it('renders input field with set width and height', () => {
  const inputField = renderer.create(
    <InputField
      width={100}
      height={20}
    />
  ).toJSON();
  expect(inputField).toMatchSnapshot();
});
