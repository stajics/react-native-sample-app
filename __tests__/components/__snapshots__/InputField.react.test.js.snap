exports[`test renders input field 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
      },
      Object {
        "marginLeft": 10,
        "marginRight": 10,
      },
      undefined,
      Object {
        "flexDirection": "row",
      },
    ]
  }>
  <TextInput
    autoCapitalize="none"
    autoCorrect={false}
    style={
      Array [
        Object {
          "backgroundColor": "white",
        },
        Object {
          "height": 40,
          "width": 400,
        },
        undefined,
        Object {
          "flex": -1,
          "textAlign": "left",
          "textAlignVertical": "top",
        },
      ]
    }
    underlineColorAndroid="transparent" />
</View>
`;

exports[`test renders input field with set width and height 1`] = `
<View
  style={
    Array [
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
      },
      Object {
        "marginLeft": 10,
        "marginRight": 10,
      },
      undefined,
      Object {
        "flexDirection": "row",
      },
    ]
  }>
  <TextInput
    autoCapitalize="none"
    autoCorrect={false}
    style={
      Array [
        Object {
          "backgroundColor": "white",
        },
        Object {
          "height": 20,
          "width": 100,
        },
        undefined,
        Object {
          "flex": -1,
          "textAlign": "left",
          "textAlignVertical": "top",
        },
      ]
    }
    underlineColorAndroid="transparent" />
</View>
`;
